lib/wor/paginate/adapters/helpers/total_count.rb

```diff
@@ -4,11 +4,10 @@ module Adapters
       module Helpers
         module TotalCount
           def total_count
-            content = @content.reorder(nil)
-            content_size = content.size
-            return content.to_a.size if content_size.is_a? Hash
+            @content_size ||= @content.reorder(nil).size
```

Comment:
Is that ok? I mean, if change the size, you can't view the new size :thinking:

Reply:
with every pagination the gem creates a new instance of the adapter